### DISCUSSION TITLE
How to reduce the docker run time

### DISCUSSION THREAD

**Discussion by shreya-dama-Crafty-Apes-VFX** [AbdBarho/stable-diffusion-webui-docker#426] [General]
I have downloaded the necessary models and saved the docker image. While trying to load that image and run it offline, it is taking too much time(~300 seconds) 

1. Is there a way I can reduce the run time?
2. What are the factors that would cause the delay?

Any suggestions and ideas would be very much appreciated.
<img width="680" alt="image" src="https://user-images.githubusercontent.com/117180689/234984157-ef5bf247-44b3-43a8-8372-23f2f0e29080.png">

**Comment by DevilaN** [comment]
Disk is SSD / HDD? Any other things?

I've got m2 SSD, 16GB RAM, 4GB vRAM and run it in docker on Linux machine. My instance is starting with lighting speeds when I compare it with yours.

    webui-docker-auto-1  | Startup time: 13.8s (import gradio: 2.2s, import ldm: 0.7s, other imports: 2.4s, list extensions: 0.3s, setup codeformer: 0.1s, load scripts: 0.7s, load SD checkpoint: 6.8s, create ui: 0.4s).

**Reply by AbdBarho** [reply]
This is because WSL2 in windows is slow. You can install docker in WSL directly without docker desktop and clone the repo to the default distribution virtual disk to make it faster, but this is not that practical.
